MIG.HomeAutomation/ZWave.cs

```diff
@@ -234,6 +234,9 @@ public bool IsConnected
 
         public object InterfaceControl(MigInterfaceCommand request)
         {
+            while(controller.CommandDelay > 0 && controller.LastCommand.AddMilliseconds(controller.CommandDelay) > DateTime.Now) { }
+            controller.LastCommand = DateTime.Now;
+
```

Comment:
ZWabeLib adds the parameters, plus delays in Discovery and Retry. This adds it to the actual device commands. Most of my scenes control more than one device, and this gives a little delay between the commands.